pkg/manager/config/config.go

```diff
@@ -579,6 +583,25 @@ func GetLogAccess() bool {
 	return ret
 }
 
+// Max allowed graphql upload size in megabytes
+func GetMaxUploadSize() int64 {
+	ret := int64(200)
+	if viper.IsSet(MaxUploadSize) {
+		ret = viper.GetInt64(MaxUploadSize)
+	}
+	return ret * 1000000
```

Comment:
to be accurate if it is megabytes (MB) it should be `1048576` (1024*1024) instead of `1000000`
sa,e with memory below

Reply:
Good point, fixed.